instrumentation/events/lttng-module/net.h

```diff
@@ -27,6 +27,110 @@ static inline unsigned char __has_network_hdr(struct sk_buff *skb)
 static struct lttng_event_field emptyfields[] = {
```

Comment:
similarly as comment for ipv4/ipv6, please review the base 10/16 formatting and ensure it is in line with the wireshark output.